testing/solomachine.go

```diff
@@ -516,11 +516,14 @@ func (solo *Solomachine) GenerateClientStateProof(clientState exported.ClientSta
 	data, err := clienttypes.MarshalClientState(solo.cdc, clientState)
 	require.NoError(solo.t, err)
 
+	merklePath := solo.GetClientStatePath(clientIDSolomachine)
```

Comment:
This testing functions in this file return a merle path with 2 elements (first one for the IBC store), I could also change those to return only the second element. So something like this for `GetClientStatePath`:

```
func (solo *Solomachine) GetClientStatePath(counterpartyClientIdentifier string) commitmenttypes.MerklePath {
  return commitmenttypes.NewMerklePath(host.FullClientStatePath(counterpartyClientIdentifier))
}
```

Although the function the becomes a one-liner, so maybe we don't even to have a separate function to construct the path. Then in the test I would get key 0, instead of 1 as the path over which the solo machine signs the bytes.

Reply:
Yes, I will do that better in a follow-up PR, otherwise if I remove these functions now that's also API breaking.